CHANGELOG.md

```diff
@@ -1,3 +1,104 @@
+# 1.8.0-rc01 (April 2025)
+
+_Changes since 1.8.0-beta02_
+
+## Highlights
+
+### Web
+
+-  Improves text input support in Safari on mobile and desktop [#1941](https://github.com/JetBrains/compose-multiplatform-core/pull/1941)
+-  Correct certain text input scenarios on Web targets [#1941](https://github.com/JetBrains/compose-multiplatform-core/pull/1941)
+New `WebTextInputService` seeks to achieve the following goals:
+1. Add support to Safari (which happens to have its own flow of input events that are inconsistent with other browsers)
+2. Correct behavior for composition and accent dialogue scenarios
+3. Introduce some changes that we will use for improving accessibility
+
+## Breaking Changes
+
+### Multiple Platforms
+
+- A custom implementation for deprecated `LocalTextInputService` is no longer supported [#1974](https://github.com/JetBrains/compose-multiplatform-core/pull/1974)
+
+## Migration Notes
+
+### Multiple Platforms
+
+- material3 library no longer adds a dependency to `material-icons-core` so if your project relied on that, you will have to explicitly add that dependency in your `build.gradle[.kts]` files: [#2025](https://github.com/JetBrains/compose-multiplatform-core/pull/2025)
+```
+implementation("org.jetbrains.compose.material:material-icons-core:1.6.11")
+```
```

Comment:
@kropp let's remove the duplicate. The next line mentions both material and material3